programmers/mock-test.js

```diff
@@ -0,0 +1,30 @@
+function solution(answers) {
+    let collect1 = 0;
+    let collect2 = 0;
+    let collect3 = 0;
+    for (let i = 0; i < answers.length; i++) {
+        if ( 수포자1[i % 5] === answers[i]) {
+            collect1 += 1;
+        }
+        if ( 수포자2[i % 8] === answers[i]) {
+            collect2 += 1;
+        }
+        if ( 수포자3[i % 10] === answers[i]) {
+            collect3 += 1;
+        }
+    }
+    const collects = [collect1, collect2, collect3];
+    const biggest = Math.max(...collects);
+    const results = [];
```

Comment:
세 명의 수포자 중 가장 문제를 많이 맞힌 사람을 찾습니다.
```suggestion
max = Math.max(count[0], count[1], count[2]);
```